tests/unit_tests/GetVariantInfoTest.php

```diff
@@ -648,7 +651,7 @@ public static function dataProviderGetVariantInfo ()
                 'position_end' => 2,
                 'type' => 'ins',
                 'warnings' => array(
-                    'WSUFFIXFORMAT' => 'The inserted sequence does not follow HGVS guidelines.',
+                    'WSUFFIXFORMAT' => 'The inserted/affected sequence does not follow HGVS guidelines.',
```

Comment:
Does it say "inserted/affected" because, in the code, you're not sure whether you're handling insertions or something else at that time? Or because you think "inserted sequence" isn't clear enough for an insertion?

Reply:
It says instered/affected because this message is now used not only for insertions, but also for suffixes for other type of variants, such as in the case of c.1_2del[A]

Reply:
Ah, clear, thanks! I'll add an `if()` then! It'll clarify things for the user and doesn't make it much harder for us.